config/settings.yml.example

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
This file is for core, the one for `smart_proxy_dynflow` (the proxy plugin itself) is in `settings.d/dynflow.yml.example`. This should be explicitly mentioned somewhere